specs/2025-02-24.yaml

```diff
@@ -1023,137 +938,445 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /v1/disbursements:
+    post:
+      summary: Create a disbursement
+      description: |
+        Create a disbursement to send money to an organization.
+      operationId: createDisbursement
+      tags:
+        - disbursements
+      security:
+        - bearerAuth: []
+      requestBody:
+        $ref: "#/components/requestBodies/CreateDisbursementRequest"
+      responses:
+        "201":
+          description: The disbursement was created
+          headers:
+            Location:
+              $ref: "#/components/headers/Location"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Disbursement"
+        "400":
+          $ref: "#/components/responses/BadRequestError"
+        "401":
+          $ref: "#/components/responses/AuthenticationError"
+        "403":
+          $ref: "#/components/responses/ForbiddenError"
+        "409":
+          $ref: "#/components/responses/ConflictError"
+    get:
+      summary: List disbursements
+      description: |
+        Returns a list of disbursements.
+      operationId: listDisbursements
+      tags:
+        - disbursements
+      security:
+        - bearerAuth: []
+      parameters:
+        - name: organization_id
+          in: query
+          description: The unique identifier for the organization
+          required: false
+          schema:
+            type: string
+          example: "org_01j8rs605a4gctmbm58d87mvsj"
+        - name: pageLimit
+          in: query
+          description: Limit the size of the list that is returned. The default (and maximum) is 100 objects.
+          required: false
+          schema:
+            type: integer
+            format: int32
+        - name: pageToken
+          in: query
```

Comment:
it's a little weird to me that these are camel case and `organization_id` is  snake case. most of our fields are snake case

Reply:
yea this is a good catch on attention to detail.. we should be using snake case. i can update

Reply:
done